addons/account/migrations/12.0.1.1/post-migration.py

```diff
@@ -0,0 +1,162 @@
+# Copyright 2018 Eficent <http://www.eficent.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openupgradelib import openupgrade
+from psycopg2.extensions import AsIs
+
+
+def map_account_payment_term_line_option(cr):
+    openupgrade.map_values(
+        cr,
+        openupgrade.get_legacy_name('option'),
+        'option',
+        [('fix_day_following_month', 'after_invoice_month'),
```

Comment:
This can't be mapped directly without setting some values in `day_of_the_month`. Please look for the semantic of each value.

Reply:
- '`fix_day_following_month`' -> '`after_invoice_month`', this map is absolute correct, they are just a rename, they do the same.
- about the other two maps, they only need to put `days` field to 31.